test/common/policy_utils.go

```diff
@@ -69,6 +69,99 @@ func PatchPlacementRule(namespace, name string) error {
 	return err
 }
 
+// ApplyPlacement function creates Placement and PlacementBinding so that it will
+// always only match the targetCluster.
+func ApplyPlacement(ctx SpecContext, namespace, policyName string) error {
+	By("Apply Placement and PlacementBinding" + namespace + "/" +
+		"placement-" + policyName + "/" + "placement-binding-" + "policyName" +
+		" with clusterSelector {name: " + ClusterNamespaceOnHub + "}")
+
+	placement := unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"apiVersion": GvrPlacement.Group + "/" + GvrPlacement.Version,
+			"kind":       "Placement",
+			"metadata": map[string]interface{}{
+				"name": "placement-" + policyName,
+			},
+			"spec": map[string]interface{}{
+				"predicates": []interface{}{
+					map[string]interface{}{
+						"requiredClusterSelector": map[string]interface{}{
+							"labelSelector": map[string]interface{}{
+								"matchExpressions": []interface{}{
+									map[string]interface{}{
+										"key":      "name",
+										"operator": "In",
+										"values": []string{
+											ClusterNamespaceOnHub,
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	_, err := ClientHubDynamic.Resource(GvrPlacement).Namespace(namespace).Create(
+		ctx, &placement, metav1.CreateOptions{},
+	)
+	if err != nil {
+		return err
+	}
+
+	placementBinding := unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"apiVersion": "policy.open-cluster-management.io/v1",
+			"kind":       "PlacementBinding",
+			"metadata": map[string]interface{}{
+				"name": "placement-binding-" + policyName,
+			},
+			"placementRef": map[string]interface{}{
+				"name":     "placement-" + policyName,
+				"kind":     "Placement",
+				"apiGroup": "cluster.open-cluster-management.io",
+			},
+			"subjects": []interface{}{
+				map[string]interface{}{
+					"name":     policyName,
+					"kind":     "Policy",
+					"apiGroup": "policy.open-cluster-management.io",
```

Comment:
This should use the GVR
```suggestion
					"apiGroup": GvrPolicy,
```